inc/compatibility/fixes/class-selfupdatingthemesfix.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Self Updating Themes Fix
+ *
+ * @package Pantheon\Compatibility\Fixes
+ */
+
+namespace Pantheon\Compatibility\Fixes;
+
+/**
+ * Self Updating Themes Fix
+ */
+class SelfUpdatingThemesFix {
+
+
+	public static function apply() {
+		/** Disable theme FTP form */
+		DefineConstantFix::apply( 'FS_METHOD', 'direct' );
```

Comment:
This is already defined elsewhere in the mu-plugin: https://github.com/pantheon-systems/pantheon-mu-plugin/blob/main/pantheon.php#L31-L42

This goes back to my comment about linking fix classes to their docs, so we can ensure the docs are up-to-date.